crates/ruff_python_ast/ast.toml

```diff
@@ -0,0 +1,174 @@
+# This file is used by generate.py to autogenerate our Python AST data model.
```

Comment:
Is it worth adding a small CI job that checks that `generated.rs` is up to date with what this script would output? Or would we just quickly get alerted by compiler errors if we forgot to run the script to update the file?

I guess maybe the more likely way that things could go wrong would be if a contributor tried to edit `generated.rs` directly and we didn't notice they were updating a generated file, and merged it. Then their edits would just get overwritten the next time somebody ran `generate.py`, and it might be pretty confusing when the tests they added started failing

Reply:
Yes good idea!  Done